terraform/modules/loadbalancer/dns.tf

```diff
@@ -0,0 +1,29 @@
+resource "aws_route53_record" "prometheus" {
```

Comment:
You might need to check that the this is being written to the correct account

Reply:
If I understand what you mean correctly, it behaves as expected. So when I deploy to mgmt-dev an A record is created for monitoring.wip...

Reply:
That sounds wrong then

Reply:
You're going to have to elaborate, because it is certainly functioning correctly from what I can see.